README.md

```diff
@@ -38,7 +38,33 @@ You may need to log out and log in to see all the changes.
 
 ## Uninstall Konsave
 To uninstall konsave, run the following:  
-`python -m pip uninstall konsave`
+`python -m pip uninstall konsave`  
+<br>
+
+---
+
+# UNRELEASED FEATURES
+These features have been recently added to Konsave but will remain unreleased for some time. The version is `2.0.0-alpha.1`. Your feedback will be appreciated greatly.
+
+## Installation of the unreleased version
+To test these features out, you can install Konsave by entering the following commands in your terminal:  
+-  `git clone https://github.com/prayag2/konsave`  
+-  `cd konsave`  
+-  `python -m pip install -e .`
```

Comment:
But overall, this also is pip-specific, rather than Konsave-specific knowledge. I'm thinking about moving this to `CONTRIBUTING.md`, since this is a type of information that would be useful to potential contributors - setting up the development environment.